engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -134,13 +140,9 @@ object ProcessCompilationError {
       WrongParameters(requiredParameters, passedParameters, nodeId.id)
   }
 
-  case class ErrorValidationParameter(validator: ParameterValidator, paramName: String, nodeId: String)
-    extends PartSubGraphCompilationError with InASingleNode
+  case class BlankParameterError(message: String, description: String, paramName: String, nodeId: String) extends ParameterValidationError
```

Comment:
It inherits from ParameterValidationError so BlankParameter / EmptyMandatoryParameter is sufficient, i think

Reply:
Changed